resources/views/contents/_stDataMarkup.blade.php

```diff
@@ -41,19 +41,13 @@
                 @endif
             }
         @endif
-    }
+    },
```

Comment:
Please make sure that it will validate with this comma at the end - when there will be no children?

Reply:
Yes, it is valid.